ddtrace/contrib/django/utils.py

```diff
@@ -100,7 +99,7 @@ def get_request_uri(request):
 
     # Build request url from the information available
     # DEV: We are explicitly omitting query strings since they may contain sensitive information
-    urlparts = dict(scheme=request.scheme, netloc=host, path=request.path, params=None, query=None, fragment=None)
+    urlparts = dict(scheme=request.scheme, netloc=host, path=request.path)
```

Comment:
```suggestion
    urlparts = {"scheme": request.scheme, "netloc": host, "path": request.path}
```
IIRC defining literal dictionaries are better performance-wise than `dict()`, might be worth changing this but not a concern

Reply:
```
In [1]: %timeit dict(one="two")                                                                                                   
163 ns ± 1.39 ns per loop (mean ± std. dev. of 7 runs, 10000000 loops each)

In [2]: %timeit {"one": "two"}                                                                                                    
41.7 ns ± 0.317 ns per loop (mean ± std. dev. of 7 runs, 10000000 loops each)
```

Reply:
👍 Yep, that's an excellent point. `dict` adds function calls so making an explicit dictionary is generally faster.